pkg/githubtemplate/github_template_test.go

```diff
@@ -116,7 +224,7 @@ func TestFind(t *testing.T) {
 				file.Close()
 			}
 
-			if got := Find(tt.args.rootDir, tt.args.name); !reflect.DeepEqual(got, tt.want) {
+			if got := FindLegacy(tt.args.rootDir, tt.args.name); *got != tt.want {
```

Comment:
How come you were able to remove `DeepEqual` comparison from here?

Reply:
Coz `tt.want` is just a string now, since we only have a since legacy template file